packages/next/src/server/lib/incremental-cache/fetch-cache.ts

```diff
@@ -182,29 +182,32 @@ export default class FetchCache implements CacheHandler {
       return
     }
 
-    try {
-      const res = await fetchRetryWithTimeout(
-        `${this.cacheEndpoint}/v1/suspense-cache/revalidate?tags=${tags
-          .map((tag) => encodeURIComponent(tag))
-          .join(',')}`,
-        {
-          method: 'POST',
-          headers: this.headers,
-          // @ts-expect-error not on public type
-          next: { internal: true },
-        }
-      )
+    for (let i = 0; i < tags.length / 64; i++) {
+      const currentTags = tags.slice(i * 64, i * 64 + 64)
```

Comment:
```suggestion
    for (let i = 0; i < Math.ceil(tags.length / 64); i++) {
      const currentTags = tags.slice(i * 64, i * 64 + 64);
```
Ensure partial last chunk is included